### ISSUE TITLE
q

### ISSUE THREAD

**Issue by slavamas** [myspaghetti/macos-virtualbox#647]
black, empty screen. macos installation never started.
[macOS_troubleshoot.txt](https://github.com/myspaghetti/macos-virtualbox/files/10856734/macOS_troubleshoot.txt)

**Comment by myspaghetti** [comment]
Hi, it appears the VM is assigned 5 CPUs. macOS doesn't like that, please assign 2 CPUs and try booting the VM again. All the downloaded files appear to have the correct MD5 sum, and all the other settings appear to be correct. The VirtualBox

**Comment by myspaghetti** [comment]
Hi, it appears the VM is assigned 5 CPUs. macOS doesn't like that, please assign 2 CPUs and try booting the VM again. All the downloaded files appear to have the correct MD5 sum, and all the other settings appear to be correct.

**Comment by slavamas** [comment]
Thank you, Jack for replying. I changed it to 2 CPUs, but it did not help -
the same black, blank screen. No GUI appears. I've been trying to run shell
script in debug mode, hoping it will tell me why it is getting stuck, but
it did not tell me much

**Comment by myspaghetti** [comment]
Make sure you change the CPU count in the VirtualBox VM settings and not just the script. Or delete the VM and let the script create a new one.

**Comment by slavamas** [comment]
Jack, it has been solved! Thank you so much for your replies and suggestions. I think it was totally my fault and I will close this issue. I am running this virtualbox remotely on other computer and accessing with "ssh -X". and it was silly of me trying to access all this in separate shell session. once I ran virtualbox & - it all have been solved and "bitten apple" was eventually installed and runs now just fine. and I ran already "delete_temporary_files". THANK YOU!
